ci-setups/pr-title-validation-github.md

```diff
@@ -0,0 +1,119 @@
+
+# Pull Request Validation CI/CD Setup on GitHub
+
+## Table of Contents
```

Comment:
For the table of contents, it would be better if you use an extension like `yzhang.markdown-all-in-one` to auto-generate the TOC with proper headings and subheadings

Reply:
Done, thanks for the recommendation, installed and configured yzhang.markdown-all-in-one.